extensions/wikia/DesignSystem/i18n/en.json

```diff
@@ -119,12 +119,12 @@
   "wds-avatar-badges-helper-tooltip": "FANDOM Helper",
   "wds-avatar-badges-discussion-moderator-tooltip": "Discussions Moderator",
   "wds-avatar-badges-staff-tooltip": "FANDOM Staff",
-  "wds-avatar-badges-vstf-tooltip": "VSTF",
+  "wds-avatar-badges-soap-tooltip": "SOAP",
```

Comment:
Shouldn't this be added in `v1` and `vstf` just removed in `v2`?

Reply:
👍🏻 fixed